public/data/glTF-projects-data.json

```diff
@@ -1833,10 +1833,11 @@
   "inputs" : [ "glTF 2.0" ]
 }, {
   "name" : "Godot Game Engine",
-  "description" : "Godot 3.0 fully supports glTF import",
+  "description" : "Version 3.2+ has support for glTF import. Version 3.4+ has support for glTF export.",
   "link" : "https://godotengine.org/",
   "type" : [ "engine" ],
-  "inputs" : [ "glTF 2.0" ]
+  "inputs" : [ "glTF 2.0" ],
+  "outputs" : [ "glTF 2.0" ]
```

Comment:
This only lists glTF versions, right? Godot does have other options for 3D model input (Collada `.dae`, FBX, and OBJ).

Reply:
It's a bit difficult to always figure that out "reliably". There are some projects where the information is listed explicitly, like https://github.com/KhronosGroup/glTF-Project-Explorer/blob/main/public/data/glTF-projects-data.json#L730 . These had often been created by looking at the project page and copy+pasting the "list of supported file formats", if they were listed. 

In many cases (like general asset libraries or 3D Tools like Blender), there is only a placeholder called `Multiple`  for different formats.

All this may be complicated a bit more when the support for certain file formats depends on the _version_.

But if you have a suggestion for what should be mentioned there for Godot, you can just add this here as a comment, and I'll integrate it. Otherwise, I'd extend the input and output to include `"Multiple"` a bit later.

Reply:
My suggestion would be to make this line `"inputs" : [ "glTF 2.0", "OBJ", "FBX", "Collada" ],` (I put them in order of what's the most preferred format to the least preferred format).

Reply:
The inputs have been updated (and I assume this are only the _inputs_ for now - the output is only glTF, and ... based on a quick search, some specific format, as of https://docs.godotengine.org/en/stable/getting_started/workflow/export/exporting_projects.html )

Reply:
That page won't have 3D model export information on it, it's about exporting games. But yes, I can confirm that OBJ, FBX, and COLLADA are exclusively import-only and Godot cannot export them. The only export 3D model format is glTF 2.0.